awx/main/scheduler/task_manager_models.py

```diff
@@ -54,14 +64,16 @@ def __contains__(self, hostname):
 class TaskManagerInstanceGroups:
     """A class representing minimal data the task manager needs to represent an InstanceGroup."""
 
-    def __init__(self, instances_by_hostname=None, instance_groups=None):
+    def __init__(self, instances_by_hostname=None, instance_groups=None, ig_qs=None):
```

Comment:
As someone unfamiliar with this part of the code I'm not really sure what `ig_qs` is representing. The `ig` part I'm assuming is instance group but what is the `qs` part, queue_size? Can we expand that part of the variable to be words to be clearer?

Reply:
queryset of instance groups. I guess maybe `instance_groups_queryset`?

Reply:
That'd be great, thanks!